core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -85,6 +86,8 @@
 
     public static final int CONTAINER_RUNNING_TIMEOUT_SEC = 30;
 
+    public static final String INTERNAL_HOST_HOSTNAME = "host.testcontainers.internal";
```

Comment:
Maybe better, to use `host.testcontainers` here?

Reply:
While the `.internal` TLD is technically 'wrong' to use, I can't actually find an official TLD that is properly reserved and semantically correct.

Given that Docker is `host.docker.internal`, I feel it's OK to follow in their tracks and keep `host.testcontainers.internal`.

Reply:
K, makes sense to be similar to Docker.